examples/cos/bucket_config/README.md

```diff
@@ -0,0 +1,86 @@
+# Bucket Configurations
+Every bucket in the IBM cloud has a configuration (like files and their permissions). Hence
+
+* You cannot create or delete a bucket configuration - you can only modify it (kind of like file permissions)
+  * you can certainly delete it from the k8s control plane
+    * hence the deletion policy <ins>should always be set to _Orphan_</ins>
+* the name  of the bucket config is the same as that of the bucket
+  * you can use either the _name_ or the _nameRef_ (the latter gets resolved to the former)
+
+(more info about IBM cloud buckets [in crossplane](../bucket/README.md), or the IBM Cloud UI site or the IBM Cloud API docs).
+
+### Discussion of the config spec 
+
+(do not bother to paste into a file - you got example files for that)
+
+The full spec is this
+
+```
+apiVersion: cos.ibmcloud.crossplane.io/v1alpha1
+kind: BucketConfig
+metadata:
+  name: configurable-1
+  annotations:
+    crossplane.io/external-name: "configurable-1"
+spec:
+  deletionPolicy: Orphan
+  forProvider:
+    name:
+    nameRef: configurable-1
+    hardQuota: 201
```

Comment:
The nameRef assumes that I have a "Bucket" object in kube with that name.  Should we mention it ?